src/xrc/src/forex.rs

```diff
@@ -964,7 +1016,7 @@ mod test {
                             class: AssetClass::FiatCurrency,
```

Comment:
You can use the USD asset defined [here](https://github.com/dfinity/exchange-rate-canister/blob/b75e5423a53721aa6b837399925047fef13535a0/src/xrc/src/api.rs#L137).

Reply:
Done for all usages in `forex.rs` and `api/test.rs`